aiven/client/cli.py

```diff
@@ -437,6 +437,14 @@ def user__tokens_expire(self) -> None:
         message = self.client.expire_user_tokens()["message"]
         print(message)
 
+    @arg.verbose
+    def user__password_change(self) -> None:
+        """Change the password of the currently logged in user"""
+        current_password = getpass.getpass("Current password:")
+        new_password = self.enter_password("New password:", confirm=True)
+        message = self.client.change_user_password(current_password, new_password)
+        print(message)
```

Comment:
In case of password reuse it print me:
```
ERROR   command failed: Error: {"errors":[{"message":"Password reuse is not allowed","status":403}],"message":"Password reuse is not allowed"}
```
Which is correct, but not user friendly. I believe:
```
ERROR   command failed: Password reuse is not allowed
```
is much more readable and enough for the user.

Reply:
On success it prints the token. For me, it's unexpected behavior. Even more: when I try to do something, it shows me:
```
ERROR   command failed: Error: {"errors":[{"message":"Expired db token","status":401}],"message":"Expired db token"}
```
It would be much easier for the user if it showed: `Password has been changed` and then did login automatically.

Reply:
I think error logging happens here https://github.com/aiven/aiven-client/blob/main/aiven/client/argx.py#L334 for all commands. I'd rather not make it different for just one command, given that I don't agree 100% with the user-friendliness argument either: shorter non-JSON message may look less intimidating, but if customer reports a failure to do something, it contains less information to enable us to resolve the problem.